CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+- possible exception in Consumermdib._get_context_states

```

Comment:
please be more specific and add the link to the issue would be nice

Reply:
I think the link to the issue is enough